tokio/tests/io_poll_aio.rs

```diff
@@ -5,7 +5,10 @@ use mio_aio::{AioFsyncMode, SourceApi};
 use std::{
     future::Future,
     io, mem,
-    os::unix::io::{AsRawFd, RawFd},
+    os::{
+        fd::AsFd,
+        unix::io::{AsRawFd, RawFd}
+    },
```

Comment:
Tokio uses this convention for imports:
```suggestion
    os::fd::AsFd,
    os::unix::io::{AsRawFd, RawFd},
```